docker-compose.yml

```diff
@@ -0,0 +1,189 @@
+version: '3.8'
+
+services:
+  angular-build:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    container_name: oppia-angular-build
+    volumes:
+      - shared_root_dir:/app/oppia
+    environment:
+      - production=false
+      - NODE_ENV=development
+      - oppia_is_dockerized=true
+    healthcheck:
+      test: test "$(ls -A /app/oppia/dist)" || exit 1
+      timeout: 30s
+      interval: 30s
+      retries: 5
+      start_period: 20m
+    entrypoint: >
+      sh -c '
+        trap "rm /app/oppia/node_modules/@angular/compiler-cli/ngcc/__ngcc_lock_file__" EXIT
+        /app/oppia/node_modules/.bin/ng build --watch
+      '
+
+  webpack-compiler:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    container_name: oppia-webpack-compiler
+    volumes:
+      - shared_root_dir:/app/oppia
+    environment:
+      - oppia_is_dockerized=true
+    healthcheck:
+      test: test "$(ls -A /app/oppia/webpack_bundles)" || exit 1
+      timeout: 30s
+      interval: 30s
+      retries: 5
+      start_period: 20m
+    entrypoint: >
+      sh -c '
+        node /app/oppia/node_modules/webpack/bin/webpack.js \
```

Comment:
yes, we can remove the `sh -c`, added the whole path because I referred from the python scripts and this was the command we currently using within the python scripts